lib/services/nodes-api-service.js

```diff
@@ -280,6 +280,11 @@ function nodeApiServiceFactory(
                     }
                 };
                 return workflowApiService.createAndRunGraph(configuration);
+            } else if (node.type === Constants.NodeTypes.Rack) {
+                return eventsProtocol.publishNodeEvent(node, 'added')
```

Comment:
I can do that, the 'added' event will only trigger for nodes created through posts to the api though.